linux_os/guide/services/obsolete/tftp/package_tftp-server_removed/rule.yml

```diff
@@ -40,6 +40,10 @@ references:
 
 {{{ complete_ocil_entry_package(package="tftp-server") }}}
 
+fixtext: '{{{ fixtext_package_removed("tftp-server") }}}'
```

Comment:
This generates quite weird text:




To remove the Red Hat Enterprise Linux 9 package tftp-server  run the following command: The  tftp-server  package can be removed with  the following command:   $ sudo dnf erase tftp-server